src/components/ui/Toast/Toaster.module.scss

```
@@ -0,0 +1,67 @@
+@use '@/styles/device.scss' as device;
+
+.viewport {
+  position: fixed;
+  left: 50%;
+  bottom: calc(#{spacing.$space-32} + env(safe-area-inset-bottom, 0px));
+  transform: translateX(-50%);
+
+  // ConfirmDialog overlay(1000) 등 page 내 fixed 요소보다 위에 떠야 함.
+  z-index: 2000;
+
+  display: flex;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)